Kickstarter-iOS/ViewModels/AppDelegateViewModelTests.swift

```diff
@@ -11,12 +11,15 @@ import UserNotifications
 import XCTest
 
 final class AppDelegateViewModelTests: TestCase {
-  let vm: AppDelegateViewModelType = AppDelegateViewModel()
+  var vm: AppDelegateViewModelType!
 
   fileprivate let applicationIconBadgeNumber = TestObserver<Int, Never>()
   fileprivate let configureAppCenterWithData = TestObserver<AppCenterConfigData, Never>()
   fileprivate let configureFabric = TestObserver<(), Never>()
+  fileprivate let configureQualtrics = TestObserver<QualtricsConfigData, Never>()
   fileprivate let didAcceptReceivingRemoteNotifications = TestObserver<(), Never>()
+  fileprivate let displayQualtricsSurvey = TestObserver<(), Never>()
+  private let evaluateQualtricsTargetingLogic = TestObserver<(), Never>()
```

Comment:
Can we make this `fileprivate` so it's consistent with the rest of the observers in this test class?

Reply:
Renamed everything to `private`, we should only use `fileprivate` when we need to.